tracer_options.go

```diff
@@ -81,7 +81,11 @@ func (tracerOptions) RandomNumber(randomNumber func() uint64) TracerOption {
 // that can access parent spans after those spans have been finished.
 func (tracerOptions) PoolSpans(poolSpans bool) TracerOption {
 	return func(tracer *Tracer) {
-		tracer.options.poolSpans = poolSpans
+		if poolSpans {
+			tracer.spanAllocator = newSpanSyncPool()
+		} else {
```

Comment:
`else` not needed, constructor defaults to this anyway

Reply:
I tried to save the current functionality of ```PoolSpans```, otherwise need to make checks inside the option that the allocators are the same what is more complicated. So I believe that current overhead easier for understanding.
In case if ```PoolSpans()``` will be called several times it won't be a big problem.